### PR TITLE
Using git to determine cortex project dir for integration tests

### PR DESCRIPTION
**Pull request by simonswine**
**What this PR does**:

Improves integration test project directory detection by using git.

**Checklist**
- ~[ ] Tests updated~
- ~[ ] Documentation added~
- ~[ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`~

### CODE REVIEW

File: integration/util.go [hunk 1]
```diff
@@ -26,6 +28,14 @@ func getCortexProjectDir() string {
 		return dir
 	}
 
+	// use the git path if available
+	if gitPath, err := exec.LookPath("git"); err == nil {
+		dir, err := exec.Command(gitPath, "rev-parse", "--show-toplevel").Output()
```
Comment (pstibrany) [inline integration/util.go:33]: Nit: no need to do `LookPath` first. exec.Command(“git”, ...) will do that already, and possibly return `exec.ErrNotFound`.

**Review by pstibrany — APPROVED** [review]
Good idea, thank you!

File: integration/util.go (hunk 1 above)
Comment (simonswine) [reply integration/util.go]: Good point, just pushed that change :+1:

**Review by stevesg — APPROVED** [review]

**Review by pracucci — APPROVED** [review]
👍  Thanks!

**Pull request merged**